src/cargo/core/compiler/fingerprint.rs

```diff
@@ -56,14 +56,14 @@
 //! Value                                      | Fingerprint | Metadata
 //! -------------------------------------------|-------------|----------
 //! rustc                                      | ✓           | ✓
-//! Profile                                    | ✓           | ✓
+//! [`Profile`]                                | ✓           | ✓
 //! `cargo rustc` extra args                   | ✓           | ✓
-//! CompileMode                                | ✓           | ✓
+//! [`CompileMode`]                            | ✓           | ✓
 //! Target Name                                | ✓           | ✓
 //! TargetKind (bin/lib/etc.)                  | ✓           | ✓
 //! Enabled Features                           | ✓           | ✓
 //! Immediate dependency’s hashes              | ✓[^1]       | ✓
-//! CompileKind (host/target)                  | ✓           | ✓
+//! [`CompileKind`] (host/target)              | ✓           | ✓
```

Comment:
tip: [`cargo clippy -W doc_markdown`](https://rust-lang.github.io/rust-clippy/master/index.html#doc_markdown) can help catch cases like this.